### ISSUE TITLE
i18n

### ISSUE THREAD

**Issue by NicolasLiampotis** [grnet/access-ca-portal#5]
Enable easy localisation for the portal with initial support for Greek and English. 

Localisable elements should be separated from the source code so that localised alternatives can be loaded or selected based on the user's preferences as needed. Localisable elements include:
- Any web page content
- Any kind of messages (Notification, Warning, Errors etc)
- E-mail notifications
- Registration Forms

**Comment by stevelaskaridis** [comment]
I think that this is more of a specification of the whole project rather than an issue, @NicolasLiampotis.
So, I would suggest it should be moved to another document, perhaps.
A rule of thumb for opting whether an addition to the project is an issue or a specification would be to consider if it can be closed or addressed on a commit/PR level. I believe this is not the case for this one.

**Comment by NicolasLiampotis** [comment]
@stevelaskaridis I agree i18n support cannot be closed by a single PR. It is however a feature (issue label updated) that can be closed once certain things are in place, e.g., Ruby I18n gem is configured, model classes/database schema are locale-aware, portal content is localised. Feel free to break it down into smaller tasks that can be linked to commits/PRs.

**Comment by stevelaskaridis** [comment]
@NicolasLiampotis 

> It is however a feature (issue label updated) that can be closed once certain things are in place.

Allow me to disagree with you on this. Let me draw a parallel for you. It's like closing an issue saying that an API should be RESTful. Of course, it can be at a certain point, but additions and changes to the codebase may bring up the same issue again and again.

**Comment by NicolasLiampotis** [comment]
When you make additions and changes to the codebase you should generally not break closed issues. So in your example, you would create a new issue in case of an API update and make sure that the API stays RESTful after resolving the new issue.

As I said, breaking i18n support into smaller issues/tasks might be useful - it is up to you.

**Comment by stevelaskaridis** [comment]
So, it's laying the foundations for I18n Internationalization (which is an issue that can be closed).
Let's agree to disagree on this one. When the time comes, don't mind if I do an "issue refactoring".

**Comment by NicolasLiampotis** [comment]
@stevelaskaridis: Feel free to do any refactoring necessary. I tried to base the current description on the i18n definition from W3C:
http://www.w3.org/International/questions/qa-i18n
